### ISSUE TITLE
The plain gui for VST3 effects is not accessible for users of screen readers 

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#2562]
Fixing this bug needs to wait until this issue is sorted out:
https://github.com/audacity/audacity/issues/2555
as fixing that may change which controls are used in the plain gui.

As it stands there are problems with the names of two types of control not being read, units not being read, and the absolute values of the parameters which use sliders not being read and indeed not being displayed for sighted keyboard users.

**Comment by DavidBailes** [comment]
Closing this, as it's already covered by:
https://github.com/audacity/audacity/issues/2313
and I've added a comment there.
